src/test/features/offer/form/models/statemanetType.ts

```diff
@@ -0,0 +1,18 @@
+import { expect } from 'chai'
+import { StatementType } from 'offer/form/models/statementType'
+
+describe('StatementType', () => {
+
+  describe('valueOf', () => {
+
+    it('should return StatementType for valid strings', () => {
+      expect(StatementType.valueOf(StatementType.ACCEPTATION.value)).to.be.equal(StatementType.ACCEPTATION)
```

Comment:
I don't see the value in checking all the types here...

Reply:
removed 2 `expect`s